tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -347,3 +347,169 @@ def test_get_manifest(self, run_command: MagicMock):
 
         run_command.side_effect = ReleaseNotFoundException()
         assert helm_wrapper.get_manifest("test-release", "test-namespace") == ()
+
+    def test_should_call_run_command_method_when_helm_template_with_api_versions(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+            api_versions="2.1.1",
+        )
+        run_command.assert_called_once_with(
+            [
+                "helm",
+                "template",
+                "test-release",
+                "bakdata-streams-bootstrap/streams-app",
+                "--api-versions" "2.1.1",
+            ],
+        )
+
+    def test_should_call_run_command_method_when_helm_template_without_api_versions(
+        self, run_command: MagicMock
+    ):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+        )
+        run_command.assert_called_once_with(
+            [
+                "helm",
+                "template",
+                "test-release",
+                "bakdata-streams-bootstrap/streams-app",
+            ],
+        )
+
+    def test_helm_template_without_api_versions(self, run_command: MagicMock):
+        helm_wrapper = Helm(helm_config=HelmConfig())
+
+        helm_wrapper.template(
+            release_name="test-release",
+            chart="bakdata-streams-bootstrap/streams-app",
+        )
+
+        run_command.return_value = """
+---
+# Source: streams-app/templates/jmx-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bakdata-streams-app-jmx-configmap
+  labels:
+    app: bakdata-streams-app
+    chart: streams-app-0.1.0
+    release: stream-bootstrap
+    heritage: Helm
```

Comment:
Can you move this YAML in a `resources` folder and read it from the file?

Reply:
Seems like a good idea. The yaml could also be generated each time with the CI so it stays up to date?

Reply:
This is not a good idea. This will make the test not reproduce able whenever the repository of streamsbootstrap updates or changes.
Besides, we don't want to test the actual functionality of the `helm template` command. We know and are sure that `helm template` will template the charts and output the Kubernetes resources.

Reply:
True, funnily enough, that test also doesn't assert anything. I will just remove it, it is wrong in multiple ways.